packages/tpetra/core/src/Tpetra_Import_def.hpp

```diff
@@ -830,466 +938,127 @@ namespace Tpetra {
     typedef Import<LO, GO, Node> import_type;
     typedef typename Array<GO>::size_type size_type;
 
-#ifdef HAVE_TPETRA_IMPORT_SETUNION_EXTRA_DEBUG_OUTPUT
-    using Teuchos::toString;
-    using std::cerr;
-    using std::endl;
-#endif // HAVE_TPETRA_IMPORT_SETUNION_EXTRA_DEBUG_OUTPUT
+#ifdef HAVE_TPETRA_MMM_TIMINGS
+    using Teuchos::TimeMonitor;
+    std::string label = std::string("Tpetra::Import::setUnion");
+    RCP<TimeMonitor> MM = rcp(new TimeMonitor(*TimeMonitor::getNewTimer(label)));
```

Comment:
Do you need the TimeMonitor to be an RCP here?  Seems like you could just use it as a scope guard.  I'm OK with leaving this to fix until next time, though :) .

Reply:
Probably not - it's a copy and paste of other `HAVE_TPETRA_MMM_TIMINGS` time monitoring code.